extensions/graphite-emitter/README.md

```diff
@@ -0,0 +1,124 @@
+
+Example `druid.historical.hist-host1_yahoo_com:8080.MyDataSourceName.GroupBy.query/time`:
+
+ * `druid` -> namespace prefix 
+ * `historical` -> service name 
+ * `hist-host1.yahoo.com:8080` -> druid hostname
+ * `MyDataSourceName` -> dimension value 
+ * `GroupBy` -> dimension value
+ * `query/time` -> metric name
+
+We have two different implementation of event converter:
+
+#### Send-All converter
+
+The first implementation called `all`, will send all the druid service metrics events. 
+The path will be in the form `<namespacePrefix>.[<druid service name>].[<druid hostname>].<dimensions values ordered by dimension's name>.<metric>`
+User has control of `<namespacePrefix>.[<druid service name>].[<druid hostname>].`
+
+You can omit the hostname by setting `ignoreHostname=true`
+`druid.SERVICE_NAME.dataSourceName.queryType.query.time`
+
+You can omit the service name by setting `ignoreServiceName=true`
+`druid.HOSTNAME.dataSourceName.queryType.query.time`
+
+```json
+
+druid.emitter.graphite.eventConverter={"type":"all", "namespacePrefix": "druid.test", "ignoreHostname":true, "ignoreServiceName":true}
+
+```
+
+#### White-list based converter
+
+The second implementation called `whiteList`, will send only the white listed metrics and dimensions.
+Same as for the `all` converter user has control of `<namespacePrefix>.[<druid service name>].[<druid hostname>].`
+White-list based converter comes with the following  default white list map
+
+```json
+{
+  "ingest/events": [],
+  "ingest/handoff/failed": [],
+  "ingest/persists": [],
+  "ingest/rows/output": [],
+  "jvm/gc": [],
+  "jvm/mem": [],
+  "query/cpu/time": [
+    "dataSource",
+    "type"
+  ],
+  "query/node/time": [
+    "dataSource",
+    "type"
+  ],
+  "query/node/ttfb": [
+    "dataSource",
+    "type"
+  ],
+  "query/partial/time": [
+    "dataSource",
+    "type"
+  ],
+  "query/segment/time": [
+    "dataSource",
+    "type"
+  ],
+  "query/segmentAndCache/time": [
+    "dataSource",
+    "type"
+  ],
+  "query/time": [
+    "dataSource",
+    "type"
+  ],
+  "query/wait/time": [
+    "dataSource",
+    "type"
+  ],
+  "segment/count": [],
+  "segment/dropQueue/count": [],
+  "segment/loadQueue/count": [],
+  "segment/loadQueue/failed": [],
+  "segment/loadQueue/size": [],
+  "segment/scan/pending": [],
+  "segment/size": [],
+  "segment/usedPercent": []
+}
+```
+
+Although user can override this by supplying a map of metrics:dimensions to `whiteListMap`
+For example the following converter will emit only the metric `query/time` with dimensions `dataSource` and `type`
+
+```json
+
+druid.emitter.graphite.eventConverter={"type":"whiteList", "namespacePrefix": "druid.test", "ignoreHostname":true, "ignoreServiceName":true, "whiteListMap":{"query/time":["dataSource","type"]}}
+
+```
+
+**Druid emits a huge number of metrics we highly recommend to use the `whiteList` converter**
```

Comment:
@himanshug thnaks please check 0ae1f37daec9d7a713f39ebdcda06228ac557471, i am not sure if it is the good way to do it